src/Test.UnitTests.Sarif/Core/WebRequestTests.cs

```diff
@@ -153,6 +157,7 @@ public void WebRequest_Parse_HandlesQueriesWithParametersWithoutValue()
         }
 
         [Fact]
+        [Trait(TestTraits.WindowsOnly, "true")]
```

Comment:
>WindowsOnly [](http://example.com/codeflow?start=26&length=11)

why do we need to mark this test as windowsonly?

Reply:
oh , the currently what I did was, I just for all those currently failed in Linux added that attribute to skip in Linux, so that we can have the build show as pass so we can check in, I have not look in each one if can be fixed, thought that can be another pr, since it is not related to my bug fix. the most basic goal was to show that my code fix and the test cases i added for it can be pass in both Windows and Linux

Reply:
fixed the code issue